src/avatar-group.scss

```diff
@@ -81,6 +81,19 @@ $block: #{$fd-namespace}-avatar-group;
     }
   }
 
+  &__overflow {
+    @include fd-reset();
+
+    &__body {
```

Comment:
results in `fd-avatar-group__overflow__body` which is not valid BEM, `fd-avatar-group__overflow-body` is fine

Reply:
@mikerodonnell89 fixed